internal/pkg/cli/task_run_test.go

```diff
@@ -444,6 +449,7 @@ func TestTaskRunOpts_Execute(t *testing.T) {
 		"error deploying resources": {
 			setupMocks: func(ctrl *gomock.Controller) {
 				mockDeployer = mocks.NewMocktaskDeployer(ctrl)
```

Comment:
Hmm i wonder why don't we follow this new unit test style? https://github.com/aws/copilot-cli/blob/master/internal/pkg/cli/env_show_test.go